gatk_base_recalibrator_4.1.8.1/example_inputs.yml

```diff
@@ -0,0 +1,58 @@
+input:
+  class: File
+  metadata: {}
+  path: input.bam
+reference:
+  class: File
+  metadata: {}
+  path: ref.fasta
+known_sites_1:
```

Comment:
very minor, but it looks like the example inputs need to be update

Reply:
Thanks